apps/www/app/examples/cards/components/github-card.tsx

```diff
@@ -38,9 +38,7 @@ export function DemoGithub() {
           <Separator orientation="vertical" className="h-[20px]" />
           <DropdownMenu>
             <DropdownMenuTrigger asChild>
-              <Button variant="secondary" className="px-2">
-                <ChevronDown className="h-4 w-4 text-secondary-foreground" />
-              </Button>
+              <ChevronDown className="h-4 w-4 cursor-pointer text-secondary-foreground" />
```

Comment:
I believe that the correct solution is not to remove the button. :thinking:

Reply:
I apologize for just seeing this message now and I've noticed that this issue seems to be unresolved. Can I still come to solve this problem?
![image](https://github.com/shadcn/ui/assets/82451257/a5c415c9-f801-45e6-b709-0cbce8ec300f)
![image](https://github.com/shadcn/ui/assets/82451257/6aaeb92f-4c9d-4453-99d5-34ce8d609ac3)